src/nvim/spell.c

```diff
@@ -2332,14 +2332,17 @@ static void spell_load_lang(char_u *lang)
 
   if (r == FAIL) {
     if (starting) {
-      // Some startup file sets &spell, but the necessary files don't exist:
-      // try to prompt the user at VimEnter. Also set spell again. #3027
-      do_cmdline_cmd(
-        "autocmd VimEnter * call spellfile#LoadFile(&spelllang)|set spell");
```

Comment:
Use of `&spelllang` caused #3966 and #4406. Need to use the first part ("en"), not the full value ("en_us").